app/pb_kits/playbook/pb_user/user.rb

```diff
@@ -37,6 +39,12 @@ def avatar_size
         end
       end
 
+      def dark_class
```

Comment:
We no longer need the underscore in `_dark` when using the new syntax of `generate_classname()`. It handles that for use. Instead this can just be "dark".  See [Body Kit,  line 33](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/pb_body/body.rb#L33) for reference